src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,37 @@
-export const Tabs = () => {};
+export const Tabs = ({ tabs, activeTabId, onTabSelected }) => {
+  const currentTab = tabs.find(tab => activeTabId === tab.id) || tabs[0];
+
+  const tabClick = tab => {
```

Comment:
Agreed, `handleTabClick ` will be better

Reply:
Fixed